ledis/Controllers/StartScreenViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  StartScreenViewController.swift
+//  ledis
+//
+//  Created by Maya Vu on 13.05.18.
+//  Copyright © 2018 Maya Vu. All rights reserved.
+//
+
+import UIKit
+
+class StartScreenViewController: UIViewController {
+    
+    @IBOutlet weak var playAsGuestButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        playAsGuestButton.layer.borderColor = UIColor.black.cgColor
+        
+        playAsGuestButton.layer.borderWidth = 0.25
+    }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: false)
```

Comment:
It's not very reliable practice to do sth with navigation controller in `viewDidDisappear `
controller should manage it in methods like `viewWillAppear `